backend/src/models/answer.js

```diff
@@ -0,0 +1,17 @@
+const db = require('./db')
+
+function getAnswers (questionID, pageNumber) {
+    // TODO - PAGING (log avoids unused variable)
+    console.log(pageNumber)
+    return new Promise((resolve, reject) => {
+        db.all(
+            'SELECT * FROM answer WHERE questionID=?',
+            [questionID],
+            (err, rows) => { err ? reject(err) : resolve(rows) }
+        )
+    })
+}
+
+module.exports = {
```

Comment:
I guess it's a question of consistency. If these types of functions are normal exports just do exports...